### PR TITLE
correctly update bounds of shapes

### PR DESCRIPTION
**Pull request by codeofsumit**

### CODE REVIEW

**Comment by codeofsumit** [comment]
fix #183

File: src/js/Edit/L.PM.Edit.Line.js [hunk 1]
```diff
@@ -181,6 +181,11 @@ Edit.Line = Edit.extend({
 
         coords.splice(index, 0, latlng);
 
+        console.log(coords);
+
+        // set new latlngs to trigger bounds update
+        this._layer.setLatLngs(coords);
```
Comment (adamtLICOR) [inline src/js/Edit/L.PM.Edit.Line.js]: 👍

**Comment by adamtLICOR** [comment]
YES!

This seems to work.

Here is the old code, pulled down via **npm**:

screen shot 2017-07-25 at 2 44 14 pm

I cloned the repo, built the `dist` folder with **node** from the command line and replaced it in my `node_modules` folder.

Annnnnd, it seems to work:

screen shot 2017-07-25 at 2 44 46 pm

Thanks again for the speedy response. I hope my fiddles and investigation were helpful.

**Comment by codeofsumit** [comment]
@adamtLICOR couldn't have fixed it without your detailed help. This is how OSS should work 😊 👍 
Thanks a lot! Will release this week but I want to see if leaflet fixes it's 1.1.0 bug that currently breaks a leaflet.pm feature.

**Comment by adamtLICOR** [comment]
Yeah, that's fine!

I'm technically only doing prototyping atm, so I can continue to use the development branch.

If something changes, I'll let you know.

**Pull request merged**